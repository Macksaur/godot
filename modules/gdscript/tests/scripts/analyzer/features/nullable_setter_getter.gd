var foo: String?:
	set(value):
		print("tried setting")
		foo = value
	get:
		print("tried getting")
		return foo

func test():
	foo = null
	print("ok" if foo == null else "nok")
