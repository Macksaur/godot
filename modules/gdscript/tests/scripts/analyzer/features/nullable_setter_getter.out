GDTEST_OK
tried setting
tried getting
ok
